src/Features/Core/Portable/Formatting/AbstractNewDocumentFormattingService.cs

```diff
@@ -42,6 +42,8 @@ public async Task<Document> FormatNewDocumentAsync(Document document, Document?
                 try
                 {
                     document = await provider.FormatNewDocumentAsync(document, hintDocument, cancellationToken).ConfigureAwait(false);
+
+                    document = await Formatter.FormatAsync(document, cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
the code is a tad confusing "FormatNewDocument" followed by "FormatAsync"... might want to clarify what's going on.

Reply:
Yes, fair point.